api/src/clojure/org/akvo/flow_api/datastore/form_instance.clj

```diff
@@ -103,9 +104,14 @@
   [_ response-str]
   response-str)
 
+(def ^DateFormat sdf (SimpleDateFormat. "dd-MM-yyyy HH:mm:ss z"))
+
 (defmethod parse-response "DATE"
   [_ response-str]
-  (ds/to-iso-8601 (java.util.Date. (Long/parseLong response-str))))
+  (let [date (try (java.util.Date. (Long/parseLong response-str))
+                  (catch NumberFormatException e
+                    (.parse sdf response-str)))]
```

Comment:
yes, we can get a `ParseException`. I don't want to swallow it and return `nil` at this point. I think it's better to wait for one and see which formats are out there.

Especially now with the quick turn-around. We get a message in sentry within a few seconds. The fix is quick and deployment automatic :)